go/host/host.go

```diff
@@ -385,8 +385,8 @@ func (h *host) waitForEnclave() {
 }
 
 // starts the host main processing loop
-func (h *host) startProcessing() {
-	h.logger.Info("§§§ Start processing...")
+// todo: matt to fix this complexity `nolint` in next PR with block provider
+func (h *host) startProcessing() { //nolint:gocognit
```

Comment:
This method gets a lot simpler with my block provider wiring PR that's in flight so I'm just ignoring the complexity warning for now.